OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -14,7 +14,7 @@ public class OSRMInstructionFormatter: Formatter {
     let version: String
     let instructions: [String: Any]
     
-    enum TokenType: String {
+    public enum TokenType: String {
```

Comment:
Note that string enumerations defined this way don’t bridge to Swift, so `string(for:modifyValueByKey:)` won’t either. You’ll have to either change `TokenType`’s raw type to `Int` and make it conform to `CustomStringConvertible`, or you’ll have to implement the string enumeration in Objective-C.